password-generator@spencerlommel.com/files/password-generator@spencerlommel.com/settings-schema.json

```diff
@@ -27,5 +34,28 @@
         "type": "checkbox",
         "default": true,
         "description": "Include Symbols"
+    },
+    "copy-settings-header" : {
+        "type" : "header",
+        "description" : "Copy Settings"
+     }, 
+    "auto-copy": {
+        "type": "checkbox",
+        "default": false,
+        "description": "Automatically Copy To Clipboard",
+        "tooltip": "When clicking the lock icon, instead of opening a window where the password can be copied it is instead automatically copied to your Clipboard."
+    },
+    "play-sound": {
+        "type": "checkbox",
+        "default": true,
+        "description": "Play sound on copy",
+        "tooltip": "Toggles whether or not a sound is played when the password is copied to the Clipboard"
+    },
+    "sound-file": {
+        "type": "soundfilechooser",
+        "default": "/usr/share/mint-artwork/sounds/map.oga",
```

Comment:
This will only be available on systems with the mint-artwork package installed, which will mostly just be Mint systems. It might be best to default to one of the freedesktop sounds so that it will work as many places as possible by default, i.e. files under `/usr/share/sounds/freedesktop/stereo/`.

Reply:
One option you have is including the sound file you want with the applet itself and then updating the path. (If you search this applets repository, you'll see other .oga and .mp3 files throughout.)

Reply:
I'll have to do the latter option and just include it in the package.

Reply:
Shout if you need a hand with the proper path and whatnot.

Reply:
Just a note that I don't have map.oga on my system so it may be better to include the audio file that you prefer to be the default so that it works universally out of the box. I'll go ahead and merge this for now but be mindful that bug is still out there.